gui/console.cpp

```diff
@@ -582,6 +591,45 @@ void ConsoleDialog::killLastWord() {
 	}
 }
 
+void ConsoleDialog::loadHistory() {
+	Common::SaveFileManager *saveFileMan = g_system->getSavefileManager();
+	Common::InSaveFile *loadFile = saveFileMan->openRawFile("history.txt");
+	if (!loadFile) {
+		debug("Could not open history.txt");
```

Comment:
This will incorrectly pop up the first time the user tries to open the console (where there's no history).

It would be better if we could check for file existence instead, and throw this if the file exists but can't be loaded

Reply:
I think I'll just remove them